<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataLinkDialog</class>
 <widget class="QDialog" name="DataLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>289</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>295</width>
    <height>289</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>连接设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/resource/Serial.png</normaloff>:/image/resource/Serial.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>串口</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>串口名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="PortNameBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="FlushButton">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>波特率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="BaudRateBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="ConnectButton">
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>局域网</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>蓝牙</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
